edge/pkg/devicetwin/dmiserver/event_type.go

```diff
@@ -0,0 +1,58 @@
+package dmiserver
+
+import (
+	"time"
+)
+
+const (
+	PropertyType string = "propertyType"
+)
+
+// MsgTwin the structure of device twin.
+type MsgTwin struct {
```

Comment:
MsgTwin -> device twin. it is strange, why not name it to deviceTwin?

Reply:
i have remove it and use the common struct from devicecontroller/types/device.go